packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -275,29 +275,24 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
       if (isEnterKeyEventFromCompositionSession(ev)) {
         return;
       }
-      if (ev.key === 'ArrowUp') {
-        /* @conditional-compile-remove(mention) */
-        if (mentionSuggestions.length > 0) {
+
+      /* @conditional-compile-remove(mention) */
+      if (mentionSuggestions.length > 0) {
+        if (ev.key === 'ArrowUp') {
           ev.preventDefault();
```

Comment:
Should we move `ev.preventDefault();` one line up so we don't need to have it in every if statement?